Add 'nativeExecution' field to 'ServerInfo' class. (#22513)
prestodb/presto: CONTRIBUTING.md

### Building the Web UI

The Presto Web UI is composed of several React components and is written in JSX and ES6. This source code is compiled and packaged into browser-compatible JavaScript, which is then checked in to the Presto source code (in the `dist` folder). You must have [Node.js](https://nodejs.org/en/download/) and [Yarn](https://yarnpkg.com/en/) installed to execute these commands.

> Install Node.js from the link above and Yarn can be installed using brew:

Reviewer: ```suggestion
Install Node.js from [Download Node.js](https://nodejs.org/en/download/)

To install Yarn using Homebrew:
```
Minor rephrasing: repeating the download link won't hurt, and suggest changing the Yarn instruction to imperative for consistency with your Node.js instruction. Also separating the two points to start the Yarn instruction on the left margin, to make it less likely for a user to overlook it.

    brew install yarn

> To update this folder after making changes, simply run:

Reviewer: ```suggestion
To update this folder after making changes, run:
```
Nit: Technical documentation should not describe something as "simple" or "easy", because "simple" and "easy" are different for different users. For more context, see Gitlab documentation style guide topic [Building Trust](https://docs.gitlab.com/ee/development/documentation/styleguide/#building-trust), and [Don’t say “simply” in your documentation by Jim Fisher](https://blog.knowledgeowl.com/blog/posts/dont-say-simply-jim-fisher/).

To iterate quickly, simply re-build the project in IntelliJ after packaging is complete. Project resources will be hot-reloaded and changes are reflected on browser refresh.

> To iterate even quicker you can use `yarn serve` to spin up web server which will track source changes and compile code for web page in real time. Assuming you are working in the `webapp` folder:

Reviewer: ```suggestion
Use `yarn serve` to spin up a web server to track source changes and compile code for web page in real time. Assuming you are working in the `webapp` folder:
```

    cd git/presto/presto-main/src/main/resources/webapp/src
    yarn serve --env=apiPort=8081

> `yarn` will show the http address to go to view the web UI (use it instead of the one from the presto server output): `http://localhost:8080/`.

Reviewer: ```suggestion
`yarn` shows the http address to view the web UI: `http://localhost:8080/`.
```
Nit: for better understanding, tell the user what they need and not about what they don't need. The "don't need" part can be a distraction.

Reviewer 2: Agree with @steveburnett 
It would be good to explain the `--env=apiPort=<port number>` and `--end=apiHost=<IP address>` more here.  The example command `yarn serve --env=apiPort=8081` may not work if the Presto server is not listening on 8081 and localhost. Since you are editing this part, could you add something like this:

Uses the following parameters to specify the Presto Server's information:
- `---env=apiHost=<IP address>`: Specify the Presto Server IP Address. The default value is `127.0.0.1`
- `---end=apiPort=<Port Number>`: Specify the Presto Server port number. The default value is `8080`
